aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -48,8 +48,6 @@ proposals:
           V3:
```

Comment:
```
V3:
  block_gas_limit: 35000
  transaction_shuffler_type:
    sender_aware_v2: 32
  transaction_deduper_type: txn_hash_and_authenticator_v1
```

Reply:
but in this way, if the first step got rejected, this one would enable the other AIP too....

Reply:
Yeah unfortunately the way the code is written I don't think there's a way to avoid this